README.md

```diff
@@ -12,19 +12,21 @@ See the support scope for [Red Hat Technology Preview](https://access.redhat.com
 ## Description
 
 The RHTPA service is the downstream redistribution of the [Trustification](https://github.com/trustification/trustification) project.
-The automation contained within this Git repository installs and configures the components of RHTPA to run on a single RHEL server, which uses a standalone containerized deployment.
-A Kubernetes-based manifest creates containers that uses [`podman kube play`](https://docs.podman.io/en/latest/markdown/podman-kube-play.1.html).
+The automation contained within this Git repository installs and configures the components of RHTPA to run on a single RHEL server, which uses a standalone containerized deployment. A Kubernetes-based manifest creates containers that uses [`podman kube play`](https://docs.podman.io/en/latest/markdown/podman-kube-play.1.html).
 
 The RHTPA Ansible collection deploys the following RHTPA components:
 
 - [Trustification](https://github.com/trustification/trustification)
 - [Guac](https://github.com/trustification/guac)
 
 An [NGINX](https://www.nginx.com) front end places an entrypoint to the RHTPA UI.
-A set of self-signed certificates get generated at runtime to establishing secure communications.
 
-The ingress host name is follow, where `<base_hostname>` is your deployment's base hostname:
-* https://`<base_hostname>`
+## Prerequisites
+
+A RHEL 9.3+ server should be used to run the Trustification components.
+
+Ansible must be installed and configured on a control node that will be used to perform the automation.
```

Comment:
This sentence is in passive voice.  All our technical documentation should be written in active voice.  I suggest rewriting this sentence, for example:

`Install and configure Ansible on a control node before performing the automated deployment.`

Reply:
Suggestion applied